ProjectManager/ProjectManager/Controller/ProjectManagerViewController.swift

```diff
@@ -0,0 +1,265 @@
+        let detailProjectViewController = DetailProjectViewController()
+        detailProjectViewController.configureEditingStatus(isEditible: true)
+        
+        detailProjectViewController.dismissHandler = { project in
+            self.projects.list.append(project)
+            self.todoTableView.reloadData()
+            self.doingTableView.reloadData()
+            self.doneTableView.reloadData()
+        }
+        
+        let navigationController = UINavigationController(rootViewController: detailProjectViewController)
+        navigationController.modalPresentationStyle = UIModalPresentationStyle.formSheet
+        
+        present(navigationController, animated: true, completion: nil)
+    }
+    
+    private func configureProjectManagerCollectionView() {
+        view.addSubview(projectManagerStackView)
+        projectManagerStackView.addArrangedSubview(todoTableView)
+        projectManagerStackView.addArrangedSubview(doingTableView)
+        projectManagerStackView.addArrangedSubview(doneTableView)
+    }
+    
+    private func configureConstraint() {
+        NSLayoutConstraint.activate([
+            projectManagerStackView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+            projectManagerStackView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor),
+            projectManagerStackView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor),
+            projectManagerStackView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor)
+        ])
+    }
+}
+
+extension ProjectManagerViewController: UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        guard let index = projectManagerStackView.arrangedSubviews.firstIndex(of: tableView),
+              let status = Status(rawValue: index) else { return 0 }
+        return projects.list.filter { $0.status == status }.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: "ProjectCell") as? ProjectCell else { return ProjectCell() }
+        
+        guard let index = projectManagerStackView.arrangedSubviews.firstIndex(of: tableView),
+              let status = Status(rawValue: index) else { return ProjectCell() }
+        
+        let assignedProjects = projects.list.filter { $0.status == status }
+        let sortedAssignedProjects = assignedProjects.sorted { $0.date > $1.date }
+        let project = sortedAssignedProjects[indexPath.row]
+        let date = project.date.formatDate()
+        
+        cell.configureContent(title: project.title, body: project.body, date: date)
+        
+        if date < Date().formatDate() {
+            cell.changeDateColor(isOverdue: true)
+        } else {
+            cell.changeDateColor(isOverdue: false)
+        }
```

Comment:
아래처럼 간단히 할 수 있을 것 같네요
```suggestion
        cell.changeDateColor(isOverdue: date < Date().formatDate())
```

Reply:
[`7bdcb08`](https://github.com/yagom-academy/ios-project-manager/pull/274/commits/7bdcb08d41c20d6c6b61729a7bae06dc2ef9d873)

코드가 짧아지고 훨씬 논리적인 방법이 된 것 같습니다! 도움을 주셔서 감사합니다!!